### ISSUE TITLE
bug when resiger models with ymal file

### ISSUE THREAD

**Issue by HuaizhengZhang** [cap-ntu/ML-Model-CI#214]
#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Problem description

when disable the convert
```
  File "/home/zhz/anaconda3/envs/modelci/lib/python3.7/site-packages/pymongo/helpers.py", line 160, in _check_command_response
    raise OperationFailure(errmsg, code, response, max_wire_version)
pymongo.errors.OperationFailure: Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed'}
```

when setting the convert as `true`

```
  File "/home/zhz/project/ML-Model-CI/modelci/hub/manager.py", line 124, in register_model
    model_input=model_input
  File "/home/zhz/project/ML-Model-CI/modelci/hub/manager.py", line 285, in _generate_model_family
    if TorchScriptConverter.from_torch_module(model, torchscript_dir):
  File "/home/zhz/project/ML-Model-CI/modelci/hub/converter.py", line 134, in from_torch_module
    model.eval()
AttributeError: 'str' object has no attribute 'eval'
```

#### Steps to Reproduce the Problem

```
source scripts/setup_env.sh

python modelci/cli/__init__.py service init

python modelci/cli/__init__.py  modelhub get -u https://s3.amazonaws.com/pytorch/models/resnet18-5c106cde.pth -p .cache/resnet18.pth


python modelci/cli/__init__.py  modelhub publish -p example/resnet50_explicit_path.yml
```

**Comment by univerone** [comment]
It's similiar to https://github.com/cap-ntu/ML-Model-CI/issues/155
Make sure the environment paramenter is set(by `source scripts/setup_env.sh`) before executing `modelci start`

**Comment by YuanmingLeee** [comment]
I think this is because the dataset, task, and metric is no more a string, pls double check @univerone

**Comment by univerone** [comment]
> I think this is because the dataset, task, and metric is no more a string, pls double check @univerone

I ran into the same error info just now, I guess it's becuase when registering a model form yml file, and then convert it, we emit model  loading process.
Besides, automatic conversion should be only permitted by PYTORCH and TRT engine or implicit engine setting instead of TorchScript Engine type.

**Comment by HuaizhengZhang** [comment]
the issue comes from the mango db verification

Once users start a new server, they should `source scripts/setup_env.sh`.

Please fix the issue after merging #215

**Comment by YuanmingLeee** [comment]
I believe this issue is fixed via #240
